src/main/scala/org/broadinstitute/dsde/workbench/leonardo/config/DataprocConfig.scala

```diff
@@ -2,7 +2,7 @@ package org.broadinstitute.dsde.workbench.leonardo.config
 
 case class DataprocConfig(serviceAccount: String,
                           dataprocInitScriptURI: String,
-                          dataprocDefaultZone: String,
+                          dataprocDefaultRegion: String,
```

Comment:
Dataproc uses regions, not zones, which are different. Changing the terminology to be consistent with Google.

https://cloud.google.com/dataproc/docs/reference/rest/v1beta2/projects.regions.clusters/create
https://cloud.google.com/compute/docs/regions-zones/regions-zones